src/components/AnonymousChat/child/ChannelTitle.tsx

```diff
@@ -18,7 +18,8 @@ const ChannelTitle = ({
   hasFollowButton = false,
   isFollowing,
   handleFollow,
-  isAnonymousTab = false
+  isAnonymousTab = false,
+  hasAttachment = false
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `hasAttachment` prop is a good way to indicate whether a message has an attachment or not. However, it's important to ensure that this prop is properly passed and updated from the parent component.
```diff
-  isAnonymousTab = false
+  isAnonymousTab = false,
+  hasAttachment = false
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->